google/cloud/storage/client.py

```diff
@@ -875,8 +875,9 @@ def create_bucket(
                 made via created bucket.
             location (str):
                 (Optional) The location of the bucket. If not passed,
-                the default location, US, will be used. See
-                https://cloud.google.com/storage/docs/bucket-locations
+                the default location, US, will be used. If specifying a dual-region,
```

Comment:
May I suggest the following:

(Optional) The location of the bucket. If not passed, the default location, 'US', will be used. If specifying a dual-region location, the regions can be specified by joining them with a plus sign, for example 'US-CENTRAL1+US-WEST1'. See:
                https://cloud.google.com/storage/docs/locations